ndm-operator.yaml

```diff
@@ -35,7 +35,8 @@ data:
         {
           "key": "os-disk-exclude-filter",
           "name": "os disk exclude filter",
-          "state": "true"
+          "state": "true",
+          "include": "/,/etc/hosts,/boot"
```

Comment:
Is using include keyword here could be confusing ?

Reply:
yes done.